test_util.py cannot be run by itself

There seem to be some test isolation problems in ```bodhi.tests.server.test_util```. If those tests are run by themselves two of them fail, but if the entire test suite is run they pass:

```
[vagrant@bodhi-dev bodhi]$ py.test bodhi/tests/server/test_util.py
<snip>
====================================================== FAILURES =======================================================
____________________________________ TestUtils.test_get_critpath_components_dummy _____________________________________

self = <bodhi.tests.server.test_util.TestUtils testMethod=test_get_critpath_components_dummy>

    def test_get_critpath_components_dummy(self):
        """ Ensure that critpath packages can be found using the hardcoded
            list.
            """
        pkgs = util.get_critpath_components()
>       assert 'kernel' in pkgs, pkgs
E       AssertionError: []
E       assert 'kernel' in []

bodhi/tests/server/test_util.py:122: AssertionError
_______________________________ TestUtils.test_get_critpath_components_not_pdc_not_rpm ________________________________

self = <bodhi.tests.server.test_util.TestUtils testMethod=test_get_critpath_components_not_pdc_not_rpm>
mock_log = <MagicMock name='log' id='140457975847440'>

    @mock.patch('bodhi.server.util.log')
    @mock.patch.dict(util.config, {
        'critpath.type': 'dummy',
    })
    def test_get_critpath_components_not_pdc_not_rpm(self, mock_log):
        """ Ensure a warning is logged when the critpath system is not pdc
            and the type of components to search for is not rpm.
            """
        pkgs = util.get_critpath_components('f25', 'module')
>       assert 'kernel' in pkgs, pkgs
E       AssertionError: []
E       assert 'kernel' in []

bodhi/tests/server/test_util.py:178: AssertionError
========================================= 2 failed, 47 passed in 3.16 seconds =========================================
```

--- a/bodhi/tests/server/test_util.py
+++ b/bodhi/tests/server/test_util.py
@@ -17,7 +17,6 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 import subprocess
-import unittest
 
 import mock
 import pkgdb2client
@@ -26,9 +25,10 @@ from bodhi.server import util
 from bodhi.server.buildsys import setup_buildsystem, teardown_buildsystem
 from bodhi.server.config import config
 from bodhi.server.models import TestGatingStatus
+from bodhi.tests.server import base
 
 
-class TestBugLink(unittest.TestCase):
+class TestBugLink(base.BaseTestCase):
     """Tests for the bug_link() function."""
     def test_short_false_with_title(self):
         """Test a call to bug_link() with short=False on a Bug that has a title."""
@@ -102,7 +102,7 @@ class TestBugLink(unittest.TestCase):
              "#1234567</a>"))
 
 
-class TestUtils(unittest.TestCase):
+class TestUtils(base.BaseTestCase):
 
     def setUp(self):
         setup_buildsystem({'buildsystem': 'dev'})
@@ -720,7 +720,7 @@ class TestUtils(unittest.TestCase):
         self.assertIn('Too many result pages, aborting at', log_debug.call_args[0][0])
 
 
-class TestCMDFunctions(unittest.TestCase):
+class TestCMDFunctions(base.BaseTestCase):
     @mock.patch('bodhi.server.log.debug')
     @mock.patch('bodhi.server.log.error')
     @mock.patch('subprocess.Popen')
@@ -776,7 +776,7 @@ class TestCMDFunctions(unittest.TestCase):
         mock_debug.assert_called_with('error')
 
 
-class TestTransactionalSessionMaker(unittest.TestCase):
+class TestTransactionalSessionMaker(base.BaseTestCase):
     """This class contains tests on the TransactionalSessionMaker class."""
     @mock.patch('bodhi.server.util.log.exception')
     @mock.patch('bodhi.server.util.Session')

